### ISSUE TITLE
No order for associations in item detail panel 

### ISSUE THREAD

**Issue by jmarks** [opensalt/opensalt#228]
When you have a lot of associations, such a "Is related to" there is not meaningful order in the list. There should be an order that is helpful such as internal associations, then external to the framework but in the server, then externally hosted. Also within a group it shoudl be in a logical order as well.

![image](https://user-images.githubusercontent.com/1319413/32237416-2267d054-be22-11e7-9ddb-035b574bee0b.png)

**Comment by roverwolf** [comment]
It looks like the javascript is not applying the association sequenceNumber ordering, nor any other ordering.

**Comment by brandonopened** [comment]
Is this related to https://github.com/opensalt/opensalt/issues/212 at all in terms of sequence number as the 'source of truth' for ordering? Should we use the same logic somehow? I feel like that wouldn't be possible.

**Comment by roverwolf** [comment]
Yes, it is sort of related.  I am not sure how much code we can currently
share between the two though.
